src/makeCalculator.js

```diff
@@ -36,8 +36,39 @@
  *
  * @return {object}
  */
-function makeCalculator() {
-  // write code here
+function makeCalculator(callback, value) {
+  const resObj = {
+    result: 0,
+    add(prevValue, number) {
+      return prevValue + number;
+    },
+
+    subtract(prevValue, number) {
+      return prevValue - number;
+    },
+
+    divide(prevValue, number) {
+      return prevValue / number;
+    },
```

Comment:
You can consider that division by zero is not allowed and return an error in such case.